### ISSUE TITLE
elastic-agent proxy_url serialization

### ISSUE THREAD

**Issue by michel-laterman** [elastic/beats#27781]
When giving a `--proxy-url` option to the agent, it is unable to read what it adds to `fleet.yml` on later restarts (or during the bootstrapping process).

For example, invoking an install command with: `--proxy-url=http://10.0.1.14:3128`
Results in the following snippit being part of `fleet.yml`
```yaml
fleet:
  output:
    elasticsearch:
      proxy_url: http://10.0.1.14:3128
```

However, when the elastic-agent is restarted, the logs contain an error:
```
could not parse configuration file /opt/Elastic/Agent/elastic-agent.yml: can not convert 'object' into 'string' accessing 'fleet.proxy_url' accessing 'fleet' accessing 'fleet'
```

For confirmed bugs, please report:
- Version: 7.14.0
- Steps to Reproduce:

1. Use a valid url for the `--proxy-url` option when enrolling/installing the agent. 
2. Restart the agent

**Comment by michel-laterman** [comment]
Duplicate of https://github.com/elastic/beats/issues/27187

**Comment by michel-laterman** [comment]
Fixed by previous issue in 7.14.1
